pkg/doc/verifiable/credential.go

```diff
@@ -402,7 +402,7 @@ func NewCredential(vcData []byte, opts ...CredentialOpt) (*Credential, []byte, e
 	crOpts := parseCredentialOpts(opts)
 
 	// Decode credential (e.g. from JWT).
-	vcDataDecoded, err := decodeRaw(vcData, crOpts)
+	vcDataDecoded, err := decodeRaw(vcData, crOpts.issuerPublicKeyFetcher)
```

Comment:
isnt issuerPublicKey is sufficient, do we need to add fetcher word ??

Reply:
It's not a public key, it's a mechanism of its retrieval based on issuer/holder id (should be DID) and key id. We get the DID and the key id from received JWS in order to load the public key and then verify the signature of JWS. The real mechanism of `PublicKeyFetcher` goes beyond the verifiable credential library. I believe you will need to implement retrieval of the key from the verifiable credential registry (see https://www.w3.org/TR/vc-data-model/#dfn-verifiable-data-registries).

Reply:
when you say 
"you will need to implement retrieval of the key from the verifiable credential registry" can you elaborate if its related to the consumption of this vc library ?

Reply:
Indirectly - yes :)
I think it will be implemented by simply DID resolution.